reference/scalability_guide/representative_catalogs.rst

```diff
@@ -52,21 +52,21 @@ How we tested?
 
 The application is installed on a server following the recommended architecture :doc:`/reference/technical_information/index`.
 
-Depending on the catalog, we use a different database storage. We install the data `fixtures` via the installer before to import the products through the default product csv import (for large product import, we split into 10 files + parallel imports + custom optimisations).
+Depending on the catalog, we used a different database storage. We installed the data `fixtures` via the installer before importing the products through the default product csv import job (for a large product import, we split the catalog into 10 files + parallel imports + custom optimisations).
 
-The targeted amount of product values imply to choose a relevant database storage, if you want to know more about how we choose the storage, please read :doc:`/reference/scalability_guide/more_than_5M_product_values`.
+The targeted amount of product values imply to choose a relevant database storage, if you want to know more about how to choose the right storage strategy, please read :doc:`/reference/scalability_guide/more_than_5M_product_values`.
```

Comment:
"The right storage method" ? 
I wouldn't say "What's our strategy.. " per say because it implies we use the PIM daily as a customer would, but that is not the case (we use it in a more advanced way). "right storage strategy" feels weird maybe yes.